dipy/segment/clusteringspeed.pyx

```diff
@@ -314,3 +885,39 @@ cdef class QuickBundles(object):
 
         """
         self.clusters.c_update(cluster_id)
+
+    def get_stats(self):
+        stats = {'nb_mdf_calls': self.stats.nb_mdf_calls,
+                 'nb_aabb_calls': self.stats.nb_aabb_calls}
+
+        return stats
+
+    cdef object _build_clustermap(self):
+        clusters = ClusterMapCentroid()
+        cdef int k
+        for k in range(self.clusters.c_size()):
+            cluster = ClusterCentroid(np.asarray(self.clusters.centroids[k].features).copy())
+            cluster.indices = np.asarray(<int[:self.clusters.clusters_size[k]]> self.clusters.clusters_indices[k]).copy()
+            clusters.add_cluster(cluster)
+
+        return clusters
+
+    def get_cluster_map(self):
+        return self._build_clustermap()
+
+
+def evaluate_aabbb_checks():
```

Comment:
Is this used somewhere? If so, there's a typo in the name -> evaluate_aa**b**bb_checks